k9mail/src/main/java/com/fsck/k9/view/RecipientSelectView.java

```diff
@@ -196,6 +197,22 @@ protected void onDetachedFromWindow() {
     }
 
     @Override
+    public void onFocusChanged(boolean hasFocus, int direction, Rect previous) {
+        super.onFocusChanged(hasFocus, direction, previous);
+        if (hasFocus) {
+            displayKeyboard();
+        }
+    }
+
+    private void displayKeyboard() {
+        InputMethodManager imm = (InputMethodManager) getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
+        if (imm == null) {
+            return;
+        }
+        imm.showSoftInput(this, InputMethodManager.SHOW_IMPLICIT);
```

Comment:
But can you get more explicit than this? It's not like there's a button users can press to show the keyboard.

A quick look at `android.widget.Editor` shows they're calling `imm.showSoftInput(mTextView, 0, null)` on focus change.